src/main/java/software/amazon/smithy/lsp/SmithyProtocolExtensions.java

```diff
@@ -0,0 +1,18 @@
+package software.amazon.smithy.lsp;
```

Comment:
Looks like this file needs a license header, which is why the builds are failing